web/src/lib/utils/asset-utils.spec.ts

```diff
@@ -98,6 +98,35 @@ describe('get file mime type', () => {
 		{ mimetype: 'image/x-sony-arw', extension: 'arw' },
 		{ mimetype: 'image/x-sony-sr2', extension: 'sr2' },
 		{ mimetype: 'image/x-sony-srf', extension: 'srf' },
+		/*** The following MIME types are allowed for upload but not returned by getFileMimeType() ***
```

Comment:
No need to have this here. Will be quite burdensome to maintain correctly.

Reply:
Fixed